app/leave-app/backend/modules/calendar_events/utils.bal

```diff
@@ -0,0 +1,162 @@
+import ballerina/lang.regexp;
+import ballerina/log;
```

Comment:
Please fix the import order.

Reply:
@suhand This happens when using the formatter. Encountered this several times. I'll manually format this.